### ISSUE TITLE
nextcloud container constantly spamming "ERROR: Wrong IP address ' '  in listen.allowed_clients"

### ISSUE THREAD

**Issue by ChojinDSL** [nextcloud/all-in-one#3491]
### Steps to reproduce
1. start the nextcloud-aio docker and look at the nextcloud-aio-nextcloud container logs

### Expected behavior 
I guess that message shouldn't show up.

### Actual behavior 
I had noticed my daily backups weren't running, so I try to trigger a manual backup via the AIO webinterface. I noticed that the master container wasn't able to shutdown the other containers. Apparently it was because the nextcloud-aio-nextcloud container was stuck in some kind of loop and just spamming "ERROR: Wrong IP address ' ' in listen.allowed_clients".
Once I manually stopped the nextcloud-aio-nextcloud container, the master container was able to shut the other containers down and initiate the backup.

I was able to start all the containers via the AIO webinterface, and nextcloud seems to be working, but I assume when it's time for the daily backup again, I will run into this issue again.

### Host OS 
Debian 12 (Bookworm)

#### Nextcloud AIO version 
Nextcloud AIO v7.4.0

#### Current channel 
beta

**Comment by ChojinDSL** [comment]
I just noticed that inside the nextcloud-aio-nextcloud container, the following is listed in /usr/local/etc/php-fpm.d/www.conf

listen.allowed_clients = 127.0.0.1,::1,172.23.0.11,,172.23.0.2,

I'm assuming that double comma between the last two IP addresses is causing the problem.

**Comment by szaimen** [comment]
Hi, thanks for the report!

> I was able to start all the containers via the AIO webinterface, and nextcloud seems to be working, 

Good to hear!

> but I assume when it's time for the daily backup again, I will run into this issue again.

I don't think that this is the reason for the mastercontainer not being able to stop the nextcloud container. I rather suspect something from the host causing the behaviour. So will be good to hear if you can reproducing it next time the daily backup runs.

**Comment by chrisborell** [comment]
I have this issue, although the AIO interface is reporting all containers up - but i get a “502 Bad Gateway“ when trying to access my next-cloud. 

No other logs except wrong ip address. This usually occurs overnight as I reinstalled yesterday and it happened last night as well.

**Comment by ebildebil** [comment]
I have run into this error as well, after switching to the beta channel today.
I see below errors:
1. Nextcloud AIO container
```
Deleting duplicate sessions
NOTICE: PHP message: 404 Not Found
Type: Slim\Exception\HttpNotFoundException

4
Message: Not found.
File: /var/www/docker-aio/php/vendor/slim/slim/Slim/Middleware/RoutingMiddleware.php
Line: 76
Trace: #0 /var/www/docker-aio/php/vendor/slim/slim/Slim/Routing/RouteRunner.php(56): Slim\Middleware\RoutingMiddleware->performRouting(Object(GuzzleHttp\Psr7\ServerRequest))
#1 /var/www/docker-aio/php/vendor/slim/csrf/src/Guard.php(476): Slim\Routing\RouteRunner->handle(Object(GuzzleHttp\Psr7\ServerRequest))
#2 /var/www/docker-aio/php/vendor/slim/slim/Slim/MiddlewareDispatcher.php(168): Slim\Csrf\Guard->process(Object(GuzzleHttp\Psr7\ServerRequest), Object(Slim\Routing\RouteRunner))
#3 /var/www/docker-aio/php/vendor/slim/twig-view/src/TwigMiddleware.php(115): Psr\Http\Server\RequestHandlerInterface@anonymous->handle(Object(GuzzleHttp\Psr7\ServerRequest))
#4 /var/www/docker-aio/php/vendor/slim/slim/Slim/MiddlewareDispatcher.php(121): Slim\Views\TwigMiddleware->process(Object(GuzzleHttp\Psr7\ServerRequest), Object(Psr\Http\Server\RequestHandlerInterface@anonymous))
#5 /var/www/docker-aio/php/src/Middleware/AuthMiddleware.php(38): Psr\Http\Server\RequestHandlerInterface@anonymous->handle(Object(GuzzleHttp\Psr7\ServerRequest))
#6 /var/www/docker-aio/php/vendor/slim/slim/Slim/MiddlewareDispatcher.php(269): AIO\Middleware\AuthMiddleware->__invoke(Object(GuzzleHttp\Psr7\ServerRequest), Object(Psr\Http\Server\RequestHandlerInterface@anonymous))
#7 /var/www/docker-aio/php/vendor/slim/slim/Slim/Middleware/ErrorMiddleware.php(76): Psr\Http\Server\RequestHandlerInterface@anonymous->handle(Object(GuzzleHttp\Psr7\ServerRequest))
#8 /var/www/docker-aio/php/vendor/slim/slim/Slim/MiddlewareDispatcher.php(121): Slim\Middleware\ErrorMiddleware->process(Object(GuzzleHttp\Psr7\ServerRequest), Object(Psr\Http\Server\RequestHandlerInterface@anonymous))
#9 /var/www/docker-aio/php/vendor/slim/slim/Slim/MiddlewareDispatcher.php(65): Psr\Http\Server\RequestHandlerInterface@anonymous->handle(Object(GuzzleHttp\Psr7\ServerRequest))
#10 /var/www/docker-aio/php/vendor/slim/slim/Slim/App.php(199): Slim\MiddlewareDispatcher->handle(Object(GuzzleHttp\Psr7\ServerRequest))
#11 /var/www/docker-aio/php/vendor/slim/slim/Slim/App.php(183): Slim\App->handle(Object(GuzzleHttp\Psr7\ServerRequest))
#12 /var/www/docker-aio/php/public/index.php(182): Slim\App->run()
#13 {main}
Tips: To display error details in HTTP response set "displayErrorDetails" to true in the ErrorHandler constructor.
```
2. Nextcloud container
```
listen.allowed_clients = 127.0.0.1,::1,172.20.0.13,,172.20.0.8,
[06-Oct-2023 21:14:25] NOTICE: fpm is running, pid 2408
[06-Oct-2023 21:14:25] NOTICE: ready to handle connections
ERROR: Wrong IP address '' in listen.allowed_clients
ERROR: Wrong IP address '' in listen.allowed_clients
ERROR: Wrong IP address '' in listen.allowed_clients
ERROR: Wrong IP address '' in listen.allowed_clients
Activating Collabora config...
Activated any config changes
ERROR: Wrong IP address '' in listen.allowed_clients
ERROR: Wrong IP address '' in listen.allowed_clients
ERROR: Wrong IP address '' in listen.allowed_clients
ERROR: Wrong IP address '' in listen.allowed_clients
ERROR: Wrong IP address '' in listen.allowed_clients
ERROR: Wrong IP address '' in listen.allowed_clients
ERROR: Wrong IP address '' in listen.allowed_clients
ERROR: Wrong IP address '' in listen.allowed_clients
ERROR: Wrong IP address '' in listen.allowed_clients
ERROR: Wrong IP address '' in listen.allowed_clients
ERROR: Wrong IP address '' in listen.allowed_clients
sh: apachectl: not found
ERROR: Wrong IP address '' in listen.allowed_clients
ERROR: Wrong IP address '' in listen.allowed_clients
```

**Comment by szaimen** [comment]
I try to reproduce the issue in the coming days. On my test instance it didnt show yet...

**Comment by ebildebil** [comment]
Thank you, in the mean while, if i switch from 'beta' to 'latest' will it switch it back to the old version. I'm fine Talk not working for a while, but would rather not take any risks with the data and backups.

**Comment by szaimen** [comment]
Yesy switching back to latest should be possible

**Comment by ebildebil** [comment]
Tried this, Getting errors in Eleastic Search and stack does not come up:

```
{"@timestamp":"2023-10-06T22:16:17.888Z", "log.level": "INFO", "message":"loaded module [x-pack-sql]", "ecs.version": "1.2.0","service.name":"ES_ECS","event.dataset":"elasticsearch.server","process.thread.name":"main","log.logger":"org.elasticsearch.plugins.PluginsService","elasticsearch.node.name":"6b4ef3be7076","elasticsearch.cluster.name":"nextcloud-aio"}
{"@timestamp":"2023-10-06T22:16:17.888Z", "log.level": "INFO", "message":"loaded module [unsigned-long]", "ecs.version": "1.2.0","service.name":"ES_ECS","event.dataset":"elasticsearch.server","process.thread.name":"main","log.logger":"org.elasticsearch.plugins.PluginsService","elasticsearch.node.name":"6b4ef3be7076","elasticsearch.cluster.name":"nextcloud-aio"}
{"@timestamp":"2023-10-06T22:16:17.888Z", "log.level": "INFO", "message":"loaded module [x-pack-async]", "ecs.version": "1.2.0","service.name":"ES_ECS","event.dataset":"elasticsearch.server","process.thread.name":"main","log.logger":"org.elasticsearch.plugins.PluginsService","elasticsearch.node.name":"6b4ef3be7076","elasticsearch.cluster.name":"nextcloud-aio"}
{"@timestamp":"2023-10-06T22:16:17.888Z", "log.level": "INFO", "message":"loaded module [runtime-fields-common]", "ecs.version": "1.2.0","service.name":"ES_ECS","event.dataset":"elasticsearch.server","process.thread.name":"main","log.logger":"org.elasticsearch.plugins.PluginsService","elasticsearch.node.name":"6b4ef3be7076","elasticsearch.cluster.name":"nextcloud-aio"}
{"@timestamp":"2023-10-06T22:16:17.889Z", "log.level": "INFO", "message":"loaded module [vector-tile]", "ecs.version": "1.2.0","service.name":"ES_ECS","event.dataset":"elasticsearch.server","process.thread.name":"main","log.logger":"org.elasticsearch.plugins.PluginsService","elasticsearch.node.name":"6b4ef3be7076","elasticsearch.cluster.name":"nextcloud-aio"}
{"@timestamp":"2023-10-06T22:16:17.889Z", "log.level": "INFO", "message":"loaded module [lang-expression]", "ecs.version": "1.2.0","service.name":"ES_ECS","event.dataset":"elasticsearch.server","process.thread.name":"main","log.logger":"org.elasticsearch.plugins.PluginsService","elasticsearch.node.name":"6b4ef3be7076","elasticsearch.cluster.name":"nextcloud-aio"}
{"@timestamp":"2023-10-06T22:16:17.889Z", "log.level": "INFO", "message":"loaded module [x-pack-eql]", "ecs.version": "1.2.0","service.name":"ES_ECS","event.dataset":"elasticsearch.server","process.thread.name":"main","log.logger":"org.elasticsearch.plugins.PluginsService","elasticsearch.node.name":"6b4ef3be7076","elasticsearch.cluster.name":"nextcloud-aio"}
{"@timestamp":"2023-10-06T22:16:19.165Z", "log.level": "INFO", "message":"using [1] data paths, mounts [[/usr/share/elasticsearch/data (/dev/nvme0n1p3)]], net usable_space [3tb], net total_space [3.4tb], types [ext4]", "ecs.version": "1.2.0","service.name":"ES_ECS","event.dataset":"elasticsearch.server","process.thread.name":"main","log.logger":"org.elasticsearch.env.NodeEnvironment","elasticsearch.node.name":"6b4ef3be7076","elasticsearch.cluster.name":"nextcloud-aio"}
{"@timestamp":"2023-10-06T22:16:19.165Z", "log.level": "INFO", "message":"heap size [512mb], compressed ordinary object pointers [true]", "ecs.version": "1.2.0","service.name":"ES_ECS","event.dataset":"elasticsearch.server","process.thread.name":"main","log.logger":"org.elasticsearch.env.NodeEnvironment","elasticsearch.node.name":"6b4ef3be7076","elasticsearch.cluster.name":"nextcloud-aio"}
{"@timestamp":"2023-10-06T22:16:19.198Z", "log.level":"ERROR", "message":"fatal exception while booting Elasticsearch", "ecs.version": "1.2.0","service.name":"ES_ECS","event.dataset":"elasticsearch.server","process.thread.name":"main","log.logger":"org.elasticsearch.bootstrap.Elasticsearch","elasticsearch.node.name":"6b4ef3be7076","elasticsearch.cluster.name":"nextcloud-aio","error.type":"java.lang.IllegalStateException","error.message":"cannot downgrade a node from version [8.10.2] to version [8.9.2]","error.stack_trace":"java.lang.IllegalStateException: cannot downgrade a node from version [8.10.2] to version [8.9.2]\n\tat org.elasticsearch.server@8.9.2/org.elasticsearch.env.NodeMetadata.verifyUpgradeToCurrentVersion(NodeMetadata.java:129)\n\tat org.elasticsearch.server@8.9.2/org.elasticsearch.env.NodeMetadata.upgradeToCurrentVersion(NodeMetadata.java:136)\n\tat org.elasticsearch.server@8.9.2/org.elasticsearch.env.NodeEnvironment.loadNodeMetadata(NodeEnvironment.java:634)\n\tat org.elasticsearch.server@8.9.2/org.elasticsearch.env.NodeEnvironment.<init>(NodeEnvironment.java:333)\n\tat org.elasticsearch.server@8.9.2/org.elasticsearch.node.Node.<init>(Node.java:490)\n\tat org.elasticsearch.server@8.9.2/org.elasticsearch.node.Node.<init>(Node.java:334)\n\tat org.elasticsearch.server@8.9.2/org.elasticsearch.bootstrap.Elasticsearch$2.<init>(Elasticsearch.java:234)\n\tat org.elasticsearch.server@8.9.2/org.elasticsearch.bootstrap.Elasticsearch.initPhase3(Elasticsearch.java:234)\n\tat org.elasticsearch.server@8.9.2/org.elasticsearch.bootstrap.Elasticsearch.main(Elasticsearch.java:72)\n"}
ERROR: Elasticsearch did not exit normally - check the logs at /usr/share/elasticsearch/logs/nextcloud-aio.log

ERROR: Elasticsearch exited unexpectedly
```

**Comment by szaimen** [comment]
Can you restore a backup from before switching to beta and check if that makea it work?

**Comment by ebildebil** [comment]
trying now

**Comment by szaimen** [comment]
This is now released with v7.4.1 Beta. Testing and feedback is welcome! See https://github.com/nextcloud/all-in-one#how-to-switch-the-channel
